examples/nodejs/README.txt

```diff
@@ -0,0 +1,3 @@
+This examples directory shows some examples written in Node.js.
+
+You can also test the command files by running from the command line.
```

Comment:
Could it also feature example command line with websocketd / node  for folks who are not sure how to combine two?